### ISSUE TITLE
how to set label smoothing in yolov8/yolov11?

### ISSUE THREAD

**Issue by xuan-xuan6** [ultralytics/yolov5#13537]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

how to set label smoothing in yolov8/yolov11?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xuan-xuan6, thank you for your interest in YOLOv5 🚀! For guidance on YOLO8 or YOLO11, we recommend referring to the official [Ultralytics Documentation](https://docs.ultralytics.com/), which covers all the latest features and configurations, including label smoothing.

If this is a custom training ❓ Question, please provide as much detail as possible, including your use case, relevant configurations, and any specific challenges you're facing. For the best results, ensure you're following the [Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) for optimal model performance.

If this is a 🐛 Bug Report, please include a **minimum reproducible example** to help us investigate the issue effectively.

## Requirements

Make sure your environment meets the following setup:
[**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) dependencies installed, including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). To set up YOLOv5:
```bash
git clone https://github.com/ultralytics/yolov5  # clone repository
cd yolov5
pip install -r requirements.txt  # install dependencies
```

## Environments

You can run YOLOv5 and other Ultralytics models in the following verified environments:
- **Notebooks** with free GPUs: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. These tests validate the correct operation of YOLOv5 and related tools.

This is an automated response to assist you as quickly as possible. An Ultralytics engineer will review your issue and provide additional support soon. Thank you for your patience! 😊

**Comment by pderrenger** [comment]
@xuan-xuan6 for YOLOv8 label smoothing: Set `label_smoothing` in your dataset YAML under `args` or pass it directly in the training command like `model.train(data=..., label_smoothing=0.1)`. See the Ultralytics YOLOv8 Training Documentation for details.

YOLO11 isn't released yet - stay tuned to our GitHub repositories for updates on new releases! 🚀

**Comment by shining-love** [comment]
> [@xuan-xuan6](https://github.com/xuan-xuan6) for YOLOv8 label smoothing: Set `label_smoothing` in your dataset YAML under `args` or pass it directly in the training command like `model.train(data=..., label_smoothing=0.1)`. See the Ultralytics YOLOv8 Training Documentation for details.
> 
> YOLO11 isn't released yet - stay tuned to our GitHub repositories for updates on new releases! 🚀

Thank you for your reply. I see that label-smoothing has been removed in the latest version of u. So, does that mean label-smoothing cannot be set in the latest version of u?

**Comment by pderrenger** [comment]
@shining-love label smoothing remains available in Ultralytics YOLOv8. You can enable it by setting the `label_smoothing` parameter in your training configuration or directly in the training command:  
```python
model.train(..., label_smoothing=0.1)
```  
For implementation details, refer to the Ultralytics YOLOv8 Training Documentation.  

YOLO11 is not yet released, but existing YOLO versions like YOLOv8 fully support label smoothing. Stay updated via our GitHub repositories! 🔍

**Comment by mikejattu** [comment]
Is there a way to apply label smoothing for Yolo11 ?

**Comment by pderrenger** [comment]
@mikejattu in current Ultralytics YOLO (including YOLO11) the `label_smoothing` option has been deprecated and stripped from the config, so there’s no built‑in switch for it anymore; if you need it with YOLO11 you’d have to fork the `ultralytics/ultralytics` repo and add it back into the loss/training code yourself (see the config reference in the [Ultralytics Docs](https://docs.ultralytics.com/reference/cfg/__init__/)).
